go.mod

```diff
@@ -100,7 +100,7 @@ require (
 	github.com/coder/quartz v0.1.2
 	github.com/coder/retry v1.5.1
 	github.com/coder/serpent v0.10.0
-	github.com/coder/terraform-provider-coder v1.0.2
+	github.com/coder/terraform-provider-coder v1.0.4
```

Comment:
Shouldn't this go to `v2.1.0`?

Reply:
@defelmnq I am certainly not familiar with the code base as you but why we are still on version 1.0.4 of provider here?